documentation/how-to/variables.en.md

```diff
@@ -1,23 +1,59 @@
 ---
-title: Global variables
+title: Variables
 tx_slug: documentation_how-to_variables
 ---
 
-# Global variables
+# Variables
 
-Variables are used in QField and QGIS to dynamically set values based on
-the current project file, the layer or the system. They can be used for
-styling, setting default values for fields and much more.
+With the support of variables you can dynamically add additional values at different levels, such as project, layer or model specific level.
+See [the relevant QGIS Documentation](https://docs.qgis.org/3.40/en/docs/user_manual/introduction/general_tools.html#storing-values-in-variables) for more information.
+They can be used for a wide range of applications, including styling, setting default field values, controlling application behavior and more.
 
+QField supports two main types of variables:
+
+- **Project Variables:** Project variables are defined and managed in your QGIS project.
+- **Global Variables:** The global variables adopt the global variables of QGIS. Additional device-specific variables can be configured directly within QField.
+
+## Project Variables
+
+Project variables are defined and managed directly within your QGIS project.
+They allow you to add custom variables that are only relevant in the context of a specific project.
+This is especially useful working among multiple users.
+
+## Global Variables
+
+A key feature in QField is the ability for users to customize variables on their devices without changing or adapting the original variables set in a project file.
+
+For example, a project could have a variable named `lang` that controls the language of labels and descriptions.
+A user can change the value of this `lang` variable on their device to their preferred language.
+These changes are saved locally and will be restored the next time they open the project, all without affecting the experience of other users sharing the same project.
```

Comment:
```suggestion
Global variables can be added or modified once and will be injected into all projects opened in QField across sessions. Global variables can be used to add a unique identifier across a fleet of QField devices for example, or determine the language for a given QField user. Since they are not bound to specific projects and are defined locally, they make for a great way to customize experience for multiple users sharing the same projects.
```